docs/ref/contrib/gis/gdal.txt

```diff
@@ -33,14 +33,18 @@ Sample Data
 The GDAL/OGR tools described here are designed to help you read in
 your geospatial data, in order for most of them to be useful you have
 to have some data to work with.  If you're starting out and don't yet
-have any data of your own to use, GeoDjango comes with a number of
-simple data sets that you can use for testing.  This snippet will
-determine where these sample files are installed on your computer::
-
-    >>> import os
-    >>> import django.contrib.gis
-    >>> GIS_PATH = os.path.dirname(django.contrib.gis.__file__)
-    >>> CITIES_PATH = os.path.join(GIS_PATH, 'tests/data/cities/cities.shp')
+have any data of your own to use, GeoDjango hosts a number of
+simple data sets that you can use for testing. This snippet will
+download one sample shapefile and one sample raster file to your computer::
+
+    >>> import os, urllib
+    >>> base_url = 'https://github.com/django/django/raw/master/tests/gis_tests/data/'
+    >>> destination = '/path/to/folder/' # Replace this with a folder on your computer
+    >>> for name in ['cities.shp', 'cities.dbf', 'cities.prj', 'cities.shx']:
+    ...     urllib.urlretrieve(base_url + 'cities/' + name, os.path.join(destination + name))
+    >>> urllib.urlretrieve(base_url + 'rasters/raster.tif', os.path.join(destination + 'raster.tif'))
+    >>> CITIES_PATH = os.path.join(destination, 'cities.shp')
+    >>> RASTER_PATH = os.path.join(destination, 'raster.tif')
```

Comment:
This is not elegant at all, but should work. Not sure if that was along the lines of what you were thinking to do.